contrib/opencensus-ext-azure/examples/metrics/batch.py

```diff
@@ -26,38 +25,43 @@
 view_manager = stats.view_manager
 stats_recorder = stats.stats_recorder
 
-# Create the measures
-# The latency in milliseconds
-m_latency_ms = measure_module.MeasureFloat(
-    "task_latency", "The task latency in milliseconds", "ms")
+COUNT_MEASURE = measure_module.MeasureInt("count",
+                                          "number count",
+                                          "number")
+COUNT_VIEW = view_module.View("count_view",
+                              "number count",
+                              ["tag1"],
+                              COUNT_MEASURE,
+                              aggregation_module.CountAggregation())
 
-# Create a view in which defines an aggregation and tag keys
-latency_view = view_module.View(
-    "task_latency_distribution",
-    "The distribution of the task latencies",
-    [],
-    m_latency_ms,
-    # Latency in buckets: [>=0ms, >=100ms, >=200ms, >=400ms, >=1s, >=2s, >=4s]
-    aggregation_module.DistributionAggregation(
-        [100.0, 200.0, 400.0, 1000.0, 2000.0, 4000.0]))
+SUM_MEASURE = measure_module.MeasureFloat("sum",
+                                          "number sum",
+                                          "number")
+SUM_VIEW = view_module.View("sum_view",
+                            "number sum",
+                            ["tag1"],
+                            SUM_MEASURE,
+                            aggregation_module.SumAggregation())
 
 
 def main():
     # Enable metrics
     # Set the interval in seconds in which you want to send metrics
-    exporter = metrics_exporter.new_metrics_exporter(export_interval=5)
+    exporter = metrics_exporter.new_metrics_exporter(
+                                export_interval=2.0,
```

Comment:
I suggest that we give a reasonable exporter interval comparable to the ingestion and processing latency.